pyface/ui/wx/action/tool_palette_manager.py

```diff
@@ -77,15 +77,15 @@ def create_tool_palette(self, parent, controller=None):
     def _add_tools(self, tool_palette, groups):
         """ Adds tools for all items in a list of groups. """
 
-        previous_non_empty_group = None
+        # previous_non_empty_group = None
         for group in self.groups:
             if len(group.items) > 0:
                 # Is a separator required?
                 ## FIXME : Does the palette need the notion of a separator?
-                ##                 if previous_non_empty_group is not None and group.separator:
-                ##                     tool_bar.AddSeparator()
+                ## if previous_non_empty_group is not None and group.separator:
+                ##     tool_bar.AddSeparator()
                 ##
-                ##                 previous_non_empty_group = group
+                ## previous_non_empty_group = group
```

Comment:
why not just delete the commented code?

Reply:
I don't have a very good reason here. It just looked like meaningful code and that there might be some value in keeping it.